src/BulletSoftBody/btSoftBody.cpp

```diff
@@ -4089,7 +4089,7 @@ void btSoftBody::defaultCollisionHandler(const btCollisionObjectWrapper* pcoWrap
             {
                 const btTransform wtr = pcoWrap->getWorldTransform();
                 const btScalar timemargin = 0;
-                const btScalar basemargin = getCollisionShape()->getMargin();
+                const btScalar basemargin = 3.0*getCollisionShape()->getMargin();
                 btVector3 mins;
```

Comment:
there are more magic constants, like 2, 3 and 4. Why are those needed? If they are important, it should be exposed to the user?

Reply:
Those are more experimental. I have removed them from this PR.